tasks/pip.yml

```diff
@@ -14,6 +26,8 @@
     virtualenv_command: "{{ services.get(item).vars.virtualenv_command|default(python_virtualenv_command) }}"
     state: "{{ services.get(item).vars.aux_pkg_state|default('latest') }}"
   loop: "{{ pacifica_enabled_services }}"
+  become: true
```

Comment:
Can we make this a variable (`pacifica_user_name_become`) in `defaults/main.yml` and default that to true?

Reply:
even if root is used, would this break anything to use become?

Reply:
No. It would only affect tasks or plays that decide to use this var to determine the desired `become_user` value. Whether `root` is used or not, it must have a defined default though someplace (i.e. `pacifica` or `root`).